library/src/scripts/styles/searchBarStyles.ts

```diff
@@ -30,7 +30,7 @@ export function searchBarVariables(theme?: object) {
     };
 
     const sizing = {
-        height: formElementVars.sizing.height,
+        height: 40,
```

Comment:
I hope that is temporary solution, I mean hardcoded `40`

Reply:
It's not a temporary solution. This is the declaration of the constant. `40` is the size specified in the mockups.